backend/app/services/scrapper/schedule_scrapper.py

```diff
@@ -47,6 +47,7 @@ def callback(ch, method, properties, body):
             period.last_update_at = now
             period.save()
             app.logger.info(f"Done scrapping kd_org: {method.routing_key}; period: {active_period}; at: {now} UTC")
+            generate_desc_prerequisite(period, username, password)
```

Comment:
@angga1518 Can you give brief documentation about this function?

Reply:
updated